Doc/library/asyncio-task.rst

```diff
@@ -254,6 +282,78 @@ Creating Tasks
       Added the *context* parameter.
 
 
+Task Groups
+===========
+
+Task groups combine a task creation API with a convenient
+and reliable API to wait for all tasks in the group to finish.
+
+Example::
```

Comment:
```suggestion
For example::
```

Since this is prose, and flows better

Reply:
Nah, I'm following the lead from the majority of examples elsewhere in this file.